### DISCUSSION TITLE
Stuck at document upload page

### DISCUSSION THREAD

**Discussion by gautamchawda** [community/community#132540] [GitHub Education]
### Hi everyone,

Page reloads when i press the process my application button , i tried to upload many times or reloaded the page many times.
![Screenshot_20240717_212806_Brave](https://github.com/user-attachments/assets/1432ca40-4c49-4d16-872f-495b5f027831)

**Comment by seungbinbaek** [comment]
저도 같은 에러에요.. 해결방법좀 누군가 찾아주세요 ㅜㅜ

**Reply by seungbinbaek** [reply]
I just tried to upload the document I just saw and it worked, so I'm sharing it. Try lowering the size to below the 1MB limit. I tried lowering it to about 400kb and it uploaded successfully!

**Reply by gautamchawda** [reply]
There is no option to upload just , camera or webcam to upload the documents. Everytime i upload and press the  process the application button it resets back to upload page

**Reply by patrickaod** [reply]
Mine keeps refreshing too!

**Reply by Adm1nofff** [reply]
Same problem and there is no possibility to ask support about that.What we need to do........+it doesn't want to open other cameras except frontal on the iphone...

**Comment by evandrojr** [comment]
Same problem

**Comment by evandrojr** [comment]
has it been fixed already?

**Comment by patrickaod** [comment]
Hey, Sorry I didn’t get back to you yesterday, was really hectic. Yes, the problem was resolved. They accepted a really blurry shot of my second screen from my webcam of my acceptance letter to the course (Insert confused emoji).  Thank you for getting back to me so quickly! I really appreciate it, I was becoming really stressed. It’s a weight off my mind for sure. If anything happens though I’ll make sure to get back in contact with you.  Thanks again, Patrick Sent from Mail for Windows From: Evandro Magalhães Leite JúniorSent: 19 July 2024 19:51To: community/communityCc: patrickaod; CommentSubject: Re: [community/community] Stuck at document upload page (Discussion #132540) has it been fixed already?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Comment by johannesnydahl** [comment]
I have the same problem. Tried with both Firefox and Chrome for Android.

**Reply by johannesnydahl** [reply]
My guess is that there is a requirement for Github to detect text in the uploaded images. 

I used v4l2loopback and ffmpeg on Linux in order to create a "virtual webcam" displaying my documents. I could therefore take images of these documents through the virtual webcam on the application website. OBS Studio's virtual cam feature can also be used. It is a really cumbersome workaround but it did the job.

GitHub, please allow us to upload digital documents :/

**Reply by johannesnydahl** [reply]
Update: Got rejected because of this workaround :(

**Comment by KhujamovCodes** [comment]
Response #1

**Reply by noderjan** [reply]
> Response #1

Thank you

**Comment by josephmatthias** [comment]
I am facing the same issue, the back camera is not working.
